reliability/single-fit/okhttp/src/test/kotlin/com/datadog/android/okhttp/HeadBasedSamplingTest.kt

```diff
@@ -81,6 +79,12 @@ class HeadBasedSamplingTest {
         mockServer.shutdown()
     }
 
+    private fun Request.Builder.parentSpan(span: DatadogSpan): Request.Builder {
+        @Suppress("UnsafeThirdPartyFunctionCall") // Span can't be null
```

Comment:
normally we don't have this check running against this module, so this `Suppress` is not really needed